main.py

```diff
@@ -1,10 +1,50 @@
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
+import json
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        info_for_players = json.load(file)
+    for name in info_for_players:
+        race_info = info_for_players.get(name).get("race")
+        Race.objects.get_or_create(
+            name=race_info.get("name"),
+            description=race_info.get("description")
+        )
```

Comment:
get_or_create returns created object. You can save the result of get_or_create to a variable and use it when you need instead of duplicating values retrieval